lib/json/serialization.nit

```diff
@@ -533,7 +533,15 @@ redef class Serializable
 		return stream.to_s
 	end
 
-	private fun accept_json_serializer(v: JsonSerializer)
+	# Refinable service to customize the serialization of this class to JSON
+	#
+	# This method can be refined to customize the serialization by either
+	# writing pure JSON directly on the stream `v.stream` or
+	# by using other services of `JsonSerializer`.
+	#
+	# Alternatively, the method `core_serialize_to` can be refined to
+	# only change the serialized attributes.
```

Comment:
Shouldn't you encourage more clearly people to redefine `core_serialize_to`?

e.g.: "Most of the time, you want to redefine `core_serialize_to` instead of this one."